lvk/HelpersImGui.cpp

```diff
@@ -54,17 +54,30 @@ layout (location = 2) in flat uint in_textureId;
 
 layout (location = 0) out vec4 out_color;
 
+layout (constant_id = 0) const uint kNonLinearColorSpace = 0;
+
 void main() {
-  out_color = in_color * texture(sampler2D(kTextures2D[in_textureId], kSamplers[0]), in_uv);
+  vec4 c = in_color * texture(sampler2D(kTextures2D[in_textureId], kSamplers[0]), in_uv);
+  if (kNonLinearColorSpace == 1) {
+    // Render UI in linear color space to sRGB framebuffer.
+    out_color = vec4(pow(c.rgb, vec3(2.2)), c.a);
+  } else {
+    out_color = c;
+  }
 })";
 
 namespace lvk {
 
 lvk::Holder<lvk::RenderPipelineHandle> ImGuiRenderer::createNewPipelineState(const lvk::Framebuffer& desc) {
+  const uint32_t nonLinearColorSpace = (ctx_.getSwapChainColorSpace() == ColorSpace_SRGB_NONLINEAR);
```

Comment:
This should be alive until the rendering pipeline exists - let's make it a member field. That's because internal `VkPipeline` objects are recreated when descriptor set layouts change.